package.json

```diff
@@ -7,12 +7,14 @@
     "@blueprintjs/core": "^3.6.1",
     "@material-ui/core": "^3.1.1",
     "gh-pages": "^2.0.0",
+    "prop-types": "^15.6.2",
     "react": "^16.5.2",
```

Comment:
keep prop-types same version as react :wink:

Reply:
@oleg-preply How? There is no such version in prop-types: https://www.npmjs.com/package/prop-types